tests/bearlib/languages/documentation/DocumentationCommentTest.py

```diff
@@ -168,3 +168,25 @@ def test_java_default(self):
                          desc='     the concatenated string\n')]]
 
         self.assertEqual(expected, parsed_docs)
+
+
+class AssembledDocumentationTest(unittest.TestCase):
+
+    def test_python_assembly(self):
```

Comment:
I think it should be inside `GeneralDocumentationCommentTest` as it's general language independent functionality^^